app/screens/flagged_posts/flagged_posts.test.js

```diff
@@ -29,20 +29,58 @@ describe('FlaggedPosts', () => {
 
     test('should match snapshot', () => {
         const wrapper = shallowWithIntl(
-            <FlaggedPosts {...baseProps}/>
+            <FlaggedPosts {...baseProps}/>,
         );
 
         expect(wrapper.getElement()).toMatchSnapshot();
     });
 
+    test('should match snapshot when getFlaggedPosts failed', async () => {
+        const error = new Error('foo');
+
+        const newProps = {
+            ...baseProps,
+            actions: {
+                ...baseProps.actions,
+                getFlaggedPosts: jest.fn(async () => ({error})),
```

Comment:
This works but I suggest to use jest convention.  Please also update other similar instances.
```suggestion
                getFlaggedPosts: jest.fn().mockResolvedValue({error}),
```